content/en/tracing/setup_overview/setup/dotnet-framework.md

```diff
@@ -219,57 +234,63 @@ To configure the Tracer using a JSON file, create `datadog.json` in the instrume
 
 {{< /tabs >}}
 
-### Configuration Variables
+## Configuration settings
 
-The following tables list the supported configuration variables. Use the first name (e.g. `DD_TRACE_AGENT_URL`) when setting environment variables or configuration files. The second name, if present (e.g. `AgentUri`), indicates the name the `TracerSettings` propery to use when changing settings in the code.
+Using the methods described above, customize your tracing configuration with these settings. Use the first name (for example, `DD_<INTEGRATION>_ENABLED`) when setting environment variables or configuration files. The second name (for example, `Enabled`), indicates the name for the `IntegrationSettings` property to use when changing settings in the code. Access these properties through the `TracerSettings.Integrations[]` indexer. 
```

Comment:
This text should be reverted to the previous text. The previous guidance was correct for everything but the integration settings (which have their own instructions later) but the new text only applies to the integration settings and is wrong for the majority of settings

Reply:
Thanks for catching this - lets revert this back to its original cc. @kayayarai